pkg/domain/storage.go

```diff
@@ -46,43 +30,11 @@ type CloudAllAssetsByTimeFetcher interface {
 	FetchAll(ctx context.Context, when time.Time, count uint, offset uint, assetType string) ([]CloudAssetDetails, error)
 }
 
-// Partition represents a database partition with the specified time range
-type Partition struct {
-	Name      string
-	CreatedAt time.Time
-	Begin     time.Time
-	End       time.Time
-	Count     int
-}
-
-// PartitionConflict is used to indicate a partition exists which overlaps with a partition requested to be created
-type PartitionConflict struct {
-	Name string
-}
-
-func (e PartitionConflict) Error() string {
-	return fmt.Sprintf("A partition already exists which overlaps with the requested partition, %s", e.Name)
-}
-
-// NotFoundPartition is used to indicate a request to delete a partition which does not exist in the db
-type NotFoundPartition struct {
-	Name string
-}
-
-func (e NotFoundPartition) Error() string {
-	return fmt.Sprintf("A partition does not exist for the requested name, %s", e.Name)
-}
-
 // EventExportHandler handles exporting a single event during export
 type EventExportHandler interface {
 	Handle(changes CloudAssetChanges) error
 }
```

Comment:
If we are getting rid of `events_local_backfill.go` can we eliminate this Handler as well? or is it used by something else?

Reply:
Will follow-up with removal, good catch.